guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -4,127 +4,12 @@
 ifdef::orcharhino[]
 Ensure the repositories required to install {ProductName} are enabled on your {EL} host.
 endif::[]
```

Comment:
Please also move this to a separate file.

Reply:
I had that at first, but since it's just a single line I questioned if it was useful. Do you have plans to expand this or do you prefer it for consistency?